web/template/about.gohtml

```diff
@@ -24,7 +24,18 @@
     </p>
     <p><b>1. Die Veranstaltung findet in einem von uns ausgestatteten Raum statt:</b><br>
         Ihre Veranstaltung läuft in einen dieser Räume:
-        <i>MI HS 1, MI HS 2, MI HS 3, 00.08.038 (Seminarraum), 00.13.009A (Seminarraum), Interims I (101), Interims I(102)</i>.<br>
+        <ul class="ml-3 pl-3 list-disc">
```

Comment:
just margin does not give me the amount of indentation, I think is needed here, this is why I use margin+padding..